web_src/js/components/RepoActionView.vue

```diff
@@ -374,6 +400,27 @@ export function initRepositoryActionView() {
   padding: 10px;
 }
 
+.job-artifacts-title {
+  font-size: 110%;
+  margin-top: 16px;
+  padding: 16px 10px 0px 20px;
+  border-top: 1px dashed var(--color-secondary);
```

Comment:
```suggestion
  border-top: 1px solid var(--color-secondary);
```

We rarely if ever have dashed borders in the UI, maybe this is better for a consistent look?